main.js

```diff
@@ -0,0 +1,54 @@
+var data = {
+  direction: 'east'
+};
+
+document.addEventListener('keydown', onArrowKeyClicks);
+var $car = document.querySelector('.car');
+
+function onArrowKeyClicks(event) {
+  if (data.direction === 'east') {
+    if (event.code === 'ArrowUp') {
+      $car.className = 'car face-north';
+      data.direction = 'north';
+    } else if (event.code === 'ArrowDown') {
+      $car.className = 'car face-south';
+      data.direction = 'south';
+    } else if (event.code === 'ArrowLeft') {
+      $car.className = 'car face-west';
+      data.direction = 'west';
+    }
+  } else if (data.direction === 'north') {
+    if (event.code === 'ArrowDown') {
+      $car.className = 'car face-south';
+      data.direction = 'south';
+    } else if (event.code === 'ArrowLeft') {
+      $car.className = 'car face-west';
+      data.direction = 'west';
+    } else if (event.code === 'ArrowRight') {
+      $car.className = 'car face-east';
+      data.direction = 'east';
+    }
+  } else if (data.direction === 'south') {
+    if (event.code === 'ArrowUp') {
+      $car.className = 'car face-north';
+      data.direction = 'north';
+    } else if (event.code === 'ArrowLeft') {
+      $car.className = 'car face-west';
+      data.direction = 'west';
+    } else if (event.code === 'ArrowRight') {
+      $car.className = 'car face-east';
+      data.direction = 'east';
+    }
+  } else if (data.direction === 'west') {
+    if (event.code === 'ArrowUp') {
+      $car.className = 'car face-north';
+      data.direction = 'north';
+    } else if (event.code === 'ArrowDown') {
+      $car.className = 'car face-south';
+      data.direction = 'south';
+    } else if (event.code === 'ArrowRight') {
+      $car.className = 'car face-east';
+      data.direction = 'east';
+    }
+  }
```

Comment:
WOAH

All the work for each of these groups of conditionals is basically the same. You should be able to have one conditional with a block for each direction, and that should be plenty.